courses/free-courses-en.md

```diff
@@ -119,6 +120,11 @@
 * [Programming Mobile Services for Android Handheld Systems: Concurrency](https://www.coursera.org/course/posaconcurrency)
 
 
+### Artificial Intelligence
+
+* [Elements of AI](https://www.elementsofai.com) - (Reaktor)
```

Comment:
- Follow creators format guidelines :wink:
- Give credit to all creators. If there are too much, `et al.` could be used as suffix to make the pointcut.

```suggestion
* [Elements of AI](https://www.elementsofai.com) - University of Helsinki, Reaktor
```